src/stream_base.cc

```diff
@@ -102,8 +102,7 @@ int StreamBase::Writev(const FunctionCallbackInfo<Value>& args) {
 
   size_t count = chunks->Length() >> 1;
 
-  uv_buf_t bufs_[16];
-  uv_buf_t* bufs = bufs_;
+  MaybeStackBuffer<uv_buf_t> bufs(count);
```

Comment:
`MaybeStackBuffer` does exactly that – it creates a bigger buffer of `count` entries when the static buffer is not large enough. :) So, yes, both a default of `16` as the second template parameter, and `count` as the second parameter.